syntax = "proto3";
package transform;
import "datastore.proto";

message TransformPayload {
  optional string string_content = 1;
  optional bytes bytes_content = 2;
  optional string json_string_content = 3;
}

message TransformResponse {
  TransformPayload result = 1;
  optional grpc.datastore.GrpcDataStoreError error = 2;
}

service Transformer {
  rpc Transform(TransformPayload) returns (TransformResponse) {}
}
